plugins/setup/src/main/java/com/wazuh/setup/index/Index.java

```diff
@@ -114,18 +115,22 @@ public void createIndex(String index) {
                         "Index created successfully: {} {}",
                         createIndexResponse.index(),
                         createIndexResponse.isAcknowledged());
+                return true;
             }
         } catch (ResourceAlreadyExistsException e) {
             log.info("Index {} already exists. Skipping.", index);
+            return true;
         }
+        return false;
```

Comment:
Can be simplified to `return true`, as in both cases the index exists:

- Did not enter the if block.
- Did enter the catch block